File: public/programs/java/p15.java
import java.util.*;
class p15
{
	public static void main(String[] args) {
		int a,b,n;
		Scanner scan=new Scanner(System.in);
		System.out.print("Enter n value: ");
		n=scan.nextInt();
		System.out.print("Enter first number: ");
		a=scan.nextInt();
		System.out.print("Enter the multiplier: ");
		b=scan.nextInt();
		System.out.print("The Geometric progression series is ");
		for(int i=0;i<n;i++)
		{
			System.out.print(a+" ");
			a=a*b;
		}
	}
}

File: public/programs/cpp/p15.cpp
// Geometric Progression

#include<iostream.h>
#include<math.h>
using namespace std;

int main()
{
	int i,n,a,r;
	cout<<"GEOMETRIC PROGRESSION....\n";
	cout<<"Enter first term:";
	cin>>a;
	cout<<"\nEnter common ratio:";
	cin>>r;
	cout<<"\nEnter n:";
	cin>>n;
	
	for(i=1;i<=n;i++)
	{
		cout<<a*(pow(r,i-1))<<" ";              //GP sequence is a,ar,ar^2,ar^3..........ar^(n-1)
		
	}
}
	